bootstrap/bootstrap.go

```diff
@@ -1412,7 +1412,12 @@ func (b *Bootstrap) defaultCheckoutPhase(ctx context.Context) error {
 			}
 		}
 
-		if err := b.shell.Run(ctx, "git", "submodule", "update", "--init", "--recursive", "--force"); err != nil {
+		args := []string{}
+		for _, config := range b.GitSubmoduleCloneConfig {
+			args = append(args, "-c", config)
+		}
+		args = append(args, []string{"submodule", "update", "--init", "--recursive", "--force"}...)
```

Comment:
```suggestion
		args = append(args, "submodule", "update", "--init", "--recursive", "--force")
```

Reply:
I should have known there would be a simpler way to do that in go 🤦‍♂️